tests/flow/test_function_calls.py

```diff
@@ -227,12 +228,27 @@ def test10_modulo_inputs(self):
         expected_result = [[0]]
         self.env.assertEquals(actual_result.result_set, expected_result)
 
-        # Validate modulo by 0
+        # Validate integer dividend modulo by 0
         query = "RETURN 3 % 0"
         try:
             actual_result = graph.query(query)
         except redis.ResponseError as e:
             self.env.assertContains("Division by zero", str(e))
+        
+        # Validate floating-point dividend modulo by 0
```

Comment:
Can you please extend the comment to include what RedisGraph returns when evaluating `1.0 % 0` ?
i.e. what would be the output on `RedisCli`

Reply:
Ok. Comments added in a3523cc.